src/batch-client/index.ts

```diff
@@ -777,6 +778,47 @@ export class ZimbraBatchClient {
 			return normalized;
 		});
 
+	public searchConversation = ({
+		conversationId,
+		query,
+		header,
+		timezone,
+		locale,
+		cursor,
+		...options
+	}: SearchConversationOptions) => {
+		let body: any = {
+			cid: conversationId,
```

Comment:
should this conversion happen using normalize@index.js#denormalize method instead of hardcoding here?